types/wlr_output.c

```diff
@@ -267,6 +267,15 @@ void wlr_output_set_subpixel(struct wlr_output *output,
 	wlr_output_schedule_done(output);
 }
 
+void wlr_output_set_format(struct wlr_output *output,
+		enum wl_shm_format format) {
+	if (output->format == format) {
```

Comment:
I was looking at the `wlr_output_set_subpixel()` function above it, and noticed it had a similar `if` statement. I assumed it was there to avoid setting `output->subpixel` to the same subpixel layout it was already set to, and maybe it would be bad to set it twice when not needed. I figured if there was a reason the `if` statement exists in `wlr_output_set_subpixel()`, then maybe it would be a good idea to also not set `output->format` twice when not needed. If there is no benefit to the `if` statement in `wlr_output_set_format()`, I can remove it.